generate_R_table.py

```diff
@@ -1,3 +1,4 @@
+#!/Users/antonmalko/anaconda/bin/ python
```

Comment:
This will only work on your computer, have you tried using the path `/usr/bin/env python`? That should work on all Unix systems. In general, keep in mind that these scripts are definitely used by people outside UMD, so we have to make sure we don't hard-code too much of our specific setup :)